src/services/ml-model/client.ts

```diff
@@ -10,8 +10,9 @@ import { MLModelService } from '../../gen/service/mlmodel/v1/mlmodel_connect';
 
 export class MLModelClient implements MLModel {
   private client: PromiseClient<typeof MLModelService>;
-  private readonly name: string;
```

Comment:
Interesting. Not sure why it was private to begin with

Reply:
Normally this would be caught because we would extend `Resource` which requires `name` to be public. But the `MLModelService` does not implement the `doCommand`, which is a requirement for a `Resource`. So the type-checker didn't yell at me when I missed it last time.